Android/app/src/main/java/io/github/project_travel_mate/utilities/DailyQuotesFragment.java

```diff
@@ -82,7 +82,6 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         mHolder.negativeButton.setOnClickListener(view1 -> doNotShowClicked());
 
         getQuote();
-
```

Comment:
remove this file from commit

Reply:
remove the DailyQuotesFragment.java file, because  I didn't add that file